pkg/kanctl/actionset.go

```diff
@@ -190,7 +191,7 @@ func newActionSet(params *PerformParams) (*crv1alpha1.ActionSet, error) {
 
 	return &crv1alpha1.ActionSet{
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: name,
```

Comment:
Should we also have a way to generate name like we have right now? It this is getting used in automation where they use same name to create actionset, that would not break.

Reply:
If actionset is being created using automation in that case users can just skip the `--name` flag, and we would generate the name for them.

Reply:
@PrasadG193 I am kueueing this.

Reply:
Cool, thanks